### ISSUE TITLE
update graph-explorer commit

### ISSUE THREAD

**Issue by BaptisteCecconi** [polyneme/topst-spase-rdf-tools#1]
I had to pull a more recent version (https://github.com/aws/graph-explorer/tree/5323b930d36fb1d304f57de3806ccc8706dd9caa) of the submodule so that the docker file compiles.

**Comment by BaptisteCecconi** [comment]
After checking with @dwinston in Madrid, I tried to rebuild the docker images from scratch:
```
docker compose build --no-cache
```
to make sure it was not using previously compiled docker images.

I still get the error, and it is a blocking error. 

```
32.62 => Installing Node.js version v16.20.2
33.43 Downloading and installing node v16.20.2...
33.84 Downloading https://nodejs.org/dist/v16.20.2/node-v16.20.2-linux-arm64.tar.gz...
######################################################################## 100.0%     
47.43 Computing checksum with sha256sum
47.45 Checksums matched!
48.23 Now using node v16.20.2 (npm v8.19.4)
48.27 Creating default alias: default -> v16.20.2 *
48.28 Failed to install Node.js v16.20.2
48.28 => Close and reopen your terminal to start using nvm or run the following to use it now:
48.28 
48.28 export NVM_DIR="$HOME/.nvm"
48.28 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
49.30 v16.20.2 is already installed.
49.48 Now using node v16.20.2 (npm v8.19.4)
49.52 default -> v16.20.2 *
49.70 Now using node v16.20.2 (npm v8.19.4)
50.35 npm WARN EBADENGINE Unsupported engine {
50.35 npm WARN EBADENGINE   package: 'pnpm@9.12.2',
50.35 npm WARN EBADENGINE   required: { node: '>=18.12' },
50.35 npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
50.35 npm WARN EBADENGINE }
51.08 
51.08 added 1 package, and audited 2 packages in 1s
51.08 
51.08 1 package is looking for funding
51.08   run `npm fund` for details
51.08 
51.08 found 0 vulnerabilities
51.08 npm notice 
51.08 npm notice New major version of npm available! 8.19.4 -> 10.9.0
51.08 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.9.0>
51.08 npm notice Run `npm install -g npm@10.9.0` to update!
51.08 npm notice 
51.11 ERROR: This version of pnpm requires at least Node.js v18.12
51.11 The current version of Node.js is v16.20.2
51.11 Visit https://r.pnpm.io/comp to see the list of past pnpm versions with respective Node.js version support.
------
failed to solve: process "/bin/sh -c yum update -y &&     yum install -y tar gzip git findutils openssl &&     mkdir -p $NVM_DIR &&     curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash &&     source $NVM_DIR/nvm.sh &&     nvm install $NODE_VERSION &&     nvm alias default $NODE_VERSION &&     nvm use $NODE_VERSION &&     npm install -g pnpm &&     pnpm install &&     yum clean all &&     yum remove -y tar gzip findutils &&     rm -rf /var/cache/yum &&     chmod a+x ./process-environment.sh" did not complete successfully: exit code: 1
```

**Comment by dwinston** [comment]
oops, updated repo README to reference *this* repo! i.e.
```sh
git clone --recurse-submodules -j8 git@github.com:polyneme/topst-spase-rdf-tools.git
```

try again :)
